Automattic/jetpack
# Update size & colors indeterminate progress bar

#### Changes proposed in this Pull Request:
The PR is a follow up for #26370, and adjusts the color and size of stripes per design. 
- Changes light color to `#F6F7F7` and dark color to `#EDEDEE`
- Adjusts the length of the stripes to `68px` which is `24px * sqrt(2) * 2` the double of longest side of isosceles right triangle. 

~~The colors used in #26370 are from the properties in sidebar of the objects in Figma, which doesn't match the real colors. Any idea? @madebynoam @ederrengifo Thanks.~~

Colors in Figma for future reference:

> in Figma, the top-most element can in some instances dictate the color its children should use and this is one of those cases since there’s an union (imagine merging a bunch of different shapes into one)
that’s why child elements can have a color that doesn’t correspond to the actual color, which is coming from the union element that “groups” them together @keoshi 

![image](https://user-images.githubusercontent.com/1425433/192406861-0550f557-7926-43a6-8a0f-fd2a1844e743.png)


#### Other information:


#### Jetpack product discussion
n/a

### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:
* Run storybook `cd projects/js-packages/storybook && npm run storybook:dev`
* Navigate to `http://localhost:50240/?path=/story/js-packages-components-indeterminate-progress-bar--default`
* Ensure it looks okay
